ci/templates/maven-test-project/pom.xml

```diff
@@ -99,10 +100,15 @@
             <version>${kotlin.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.jetbrains.compose.material3</groupId>
-            <artifactId>material3-desktop</artifactId>
+            <groupId>org.jetbrains.compose.foundation</groupId>
```

Comment:
After including foundation, we have another issue, which we missed because material3 included only 1.8.2: TODO